package.json

```diff
@@ -43,8 +44,8 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "frontmatter-markdown-loader": "3.6.3",
-    "husky": ">=6",
-    "lint-staged": ">=10",
+    "husky": "7.0.1",
+    "lint-staged": "11.0.1",
```

Comment:
I also updated these dependencies to its exact versions rather than using `>=` notations.